src/compiled.h

```diff
@@ -87,32 +87,35 @@ typedef UInt    RNam;
 
 /* checks, should go into 'gap.c'  * * * * * * * * * * * * * * * * * * * * */
 
-#define CHECK_BOUND(obj,name) \
- if ( obj == 0 ) ErrorQuitBound(name);
+#define CHECK_BOUND(obj, name)                                               \
```

Comment:
This change doesn't need to be made, but I wonder if all these could just be functions instead of macros?